Automattic/wp-calypso
# Show ratings for paid plugins

#### Proposed Changes

* removes guard to show ratings for marketplace products
* removes scaling of ratings for WPCOM results, as we've already adjusted the scores on the backend

#### Testing Instructions



* Visit `/plugins`
* Verify that ratings are shown for paid products in the featured sections
* Go to `/plugins/browse/paid`
* Verify that ratings are shown for most products (where we have the data)
* Search for a paid plugin (eg. `woocommerce bookings`)
* Verify that both the rating and the number of ratings are shown on the search result (where we have the data)

Page | Before | After
--- | ------ | ----- 
Paid section | <img width="1057" alt="CleanShot 2023-01-23 at 16 08 58@2x" src="https://user-images.githubusercontent.com/11555574/214091610-ce679354-a850-4c24-976e-041474494b01.png"> | <img width="1065" alt="CleanShot 2023-01-23 at 16 08 09@2x" src="https://user-images.githubusercontent.com/11555574/214091662-0169ca0c-c703-419a-8d67-5dff2696d2fd.png">
Paid page | <img width="1045" alt="CleanShot 2023-01-23 at 16 19 30@2x" src="https://user-images.githubusercontent.com/11555574/214091958-d28e0565-f883-491d-8531-44889d815381.png"> | <img width="1048" alt="CleanShot 2023-01-23 at 16 08 33@2x" src="https://user-images.githubusercontent.com/11555574/214091821-ec54a7ca-7cc2-4c68-879e-25f510196cf5.png">
Search | <img width="1051" alt="CleanShot 2023-01-23 at 16 10 03@2x" src="https://user-images.githubusercontent.com/11555574/214089912-4eb469fc-d4b7-44b8-937a-87f7d2d06fc8.png"> | <img width="1047" alt="CleanShot 2023-01-23 at 16 09 48@2x" src="https://user-images.githubusercontent.com/11555574/214089989-55ae6e07-9135-44b8-b68a-d5840a5db7b9.png"> |

#### Pre-merge Checklist




Fixes #72382
